tests/codex/helpers.nim

```diff
@@ -37,28 +37,24 @@ proc corruptBlocks*(
   manifest: Manifest,
   blks, bytes: int): Future[seq[int]] {.async.} =
   var pos: seq[int]
-  while true:
-    if pos.len >= blks:
-      break
 
-    var i = -1
-    if (i = Rng.instance.rand(manifest.len - 1); pos.find(i) >= 0):
+  doAssert blks < manifest.len
```

Comment:
The changes in this file are not necessary (just some cleanup I did while troubleshooting), but still nice IMO

Reply:
👍 that's how I understood them.